babel.config.js

```diff
@@ -12,6 +13,38 @@ const targets = isBrowser
 	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
 	: { node: 'current' };
 
+const extensionOverrides = [
+	[
+		'@wordpress/import-jsx-pragma',
+		{
+			scopeVariable: 'createElement',
+			source: '@wordpress/element',
+			isDefault: false,
+		},
+	],
+	[
+		'@babel/transform-react-jsx',
+		{
+			pragma: 'createElement',
+		},
+	],
+];
+
+// The output directory is set when SDK runs, so we only set the POT generator override in that case.
+if ( outputDir ) {
```

Comment:
Another (maybe easier to read since code stays in one place?) way to structure this would be by using [`lodash.compact`](https://lodash.com/docs/4.17.11#compact):

```js
plugins: compact( [
	outputDir && [
		'@wordpress/babel-plugin-makepot',
		{
        ...
        }
	],
	[
		'@wordpress/import-jsx-pragma',
		{
			scopeVariable: 'createElement',
			source: '@wordpress/element',
			isDefault: false,
		},
	],
	[
		'@babel/transform-react-jsx',
		{
			pragma: 'createElement',
		},
	],
] )
```

You can see similar pattern here:
https://github.com/Automattic/wp-calypso/blob/ecf929d1b98243cf739596f9c382349677d8179d/webpack.config.js#L312-L313